index.html

```diff
@@ -383,21 +401,26 @@ <h5>Step 2 : Development</h5>
                                     <h5>Step 3 : Support and maintenance</h5>
                                 </div>
                                 <div class="content_area">
-                                    1 year warranty, follow backport
-                                    <p>Huic quum fas cui suum sunt omne ego. Sed toga rea vim eram tur quum. Adjuvetis perlegere immittant quaslibet separatum nec eas rum uno explorant. Eos mox tur tum casu nunc huic bile meum. Ad sequentia expendere considero se dubitandi. Loco modi regi tur sex apti data olim. </p>
+                                    <p>
+                                    We offer a year warrantly on our ports and we also do backporting.
```

Comment:
warranty

Reply:
My proposition : 

After the initial release, we support the game for a full-year, under warranty. This covers bugs and platforms' requirement changes.

We also integrates all the changes made on the main target after release